packages/hoppscotch-common/src/components/share/templates/Embeds.vue

```diff
@@ -98,5 +84,19 @@ const noActiveTab = computed(() => {
   return embedOptions.value.tabs.every((tab) => !tab.enabled)
 })
 
-const isEmbedThemeLight = computed(() => embedOptions.value.theme === "light")
+const systemTheme = useSetting("BG_COLOR")
```

Comment:
This will result in the embed theme being based on the one set on the settings page. Ideally, it should follow the one set at system preferences. Tracked in [HFE-355](https://linear.app/hopp/issue/HFE-355/the-system-theme-for-embeds-follows-the-theme-on-the-settings-page).